Catgur will crash the bot if there is no images in the subreddit on imgur.

Trace back:

```
2014/12/12 00:02:03 #cats : scottjab :  !bacon
2014/12/12 00:02:03 Command:  bacon
panic: invalid argument to Intn

goroutine 20 [running]:
runtime.panic(0x619380, 0xc2082f3a10)
    /usr/lib/golang/src/pkg/runtime/panic.c:279 +0xf5
math/rand.(*Rand).Intn(0xc208000160, 0xffffffffffffffff, 0xc208301090)
    /usr/lib/golang/src/pkg/math/rand/rand.go:95 +0x71
math/rand.Intn(0xffffffffffffffff, 0x1)
    /usr/lib/golang/src/pkg/math/rand/rand.go:195 +0x34
_/home/scottjab/gocode/src/github.com/scottjab/catbot/plugins/randgur.randInt(0x0, 0xffffffffffffffff, 0xd)
    /home/scottjab/gocode/src/github.com/scottjab/catbot/plugins/randgur/randgur.go:17 +0x2f
_/home/scottjab/gocode/src/github.com/scottjab/catbot/plugins/randgur.RandomImageFromSubReddit(0x6e7c90, 0xd, 0x0, 0x0)
    /home/scottjab/gocode/src/github.com/scottjab/catbot/plugins/randgur/randgur.go:29 +0x297
_/home/scottjab/gocode/src/github.com/scottjab/catbot/plugins.Handler(0xc208004180)
    /home/scottjab/gocode/src/github.com/scottjab/catbot/plugins/catgur.go:33 +0x196
created by main.main
    /home/scottjab/gocode/src/github.com/scottjab/catbot/main.go:65 +0xf5

goroutine 16 [chan receive, 3 minutes]:
main.main()
    /home/scottjab/gocode/src/github.com/scottjab/catbot/main.go:94 +0x329

goroutine 19 [finalizer wait, 6 minutes]:
runtime.park(0x414750, 0x85d510, 0x85b769)
    /usr/lib/golang/src/pkg/runtime/proc.c:1369 +0x89
runtime.parkunlock(0x85d510, 0x85b769)
    /usr/lib/golang/src/pkg/runtime/proc.c:1385 +0x3b
runfinq()
    /usr/lib/golang/src/pkg/runtime/mgc0.c:2644 +0xcf
runtime.goexit()
    /usr/lib/golang/src/pkg/runtime/proc.c:1445

goroutine 17 [syscall, 6 minutes]:
runtime.goexit()
    /usr/lib/golang/src/pkg/runtime/proc.c:1445

goroutine 21 [select]:
github.com/fluffle/goirc/client.(*Conn).send(0xc2080275f0)
    /home/scottjab/gocode/src/github.com/fluffle/goirc/client/connection.go:259 +0x11c
created by github.com/fluffle/goirc/client.(*Conn).postConnect
    /home/scottjab/gocode/src/github.com/fluffle/goirc/client/connection.go:240 +0x131

goroutine 22 [IO wait]:
net.runtime_pollWait(0x7f7551ecc718, 0x72, 0x0)
    /usr/lib/golang/src/pkg/runtime/netpoll.goc:146 +0x66
net.(*pollDesc).Wait(0xc208028060, 0x72, 0x0, 0x0)
    /usr/lib/golang/src/pkg/net/fd_poll_runtime.go:84 +0x46
net.(*pollDesc).WaitRead(0xc208028060, 0x0, 0x0)
    /usr/lib/golang/src/pkg/net/fd_poll_runtime.go:89 +0x42
net.(*netFD).Read(0xc208028000, 0xc20829c000, 0x2000, 0x2000, 0x0, 0x7f7551ecb3f0, 0xb)
    /usr/lib/golang/src/pkg/net/fd_unix.go:242 +0x34c
net.(*conn).Read(0xc20803a030, 0xc20829c000, 0x2000, 0x2000, 0x0, 0x0, 0x0)
    /usr/lib/golang/src/pkg/net/net.go:122 +0xe7
crypto/tls.(*block).readFromUntil(0xc2082fab40, 0x7f7551ecc7f0, 0xc20803a030, 0x5, 0x0, 0x0)
    /usr/lib/golang/src/pkg/crypto/tls/conn.go:451 +0xd9
crypto/tls.(*Conn).readRecord(0xc208024000, 0x17, 0x0, 0x0)
    /usr/lib/golang/src/pkg/crypto/tls/conn.go:536 +0x1ff
crypto/tls.(*Conn).Read(0xc208024000, 0xc208304000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
    /usr/lib/golang/src/pkg/crypto/tls/conn.go:901 +0x16a
bufio.(*Reader).fill(0xc208004ea0)
    /usr/lib/golang/src/pkg/bufio/bufio.go:97 +0x1b3
bufio.(*Reader).ReadSlice(0xc208004ea0, 0xc2080a250a, 0x0, 0x0, 0x0, 0x0, 0x0)
    /usr/lib/golang/src/pkg/bufio/bufio.go:298 +0x22c
bufio.(*Reader).ReadBytes(0xc208004ea0, 0xa, 0x0, 0x0, 0x0, 0x0, 0x0)
    /usr/lib/golang/src/pkg/bufio/bufio.go:376 +0xe3
bufio.(*Reader).ReadString(0xc208004ea0, 0xc20806400a, 0x0, 0x0, 0x0, 0x0)
    /usr/lib/golang/src/pkg/bufio/bufio.go:416 +0x58
github.com/fluffle/goirc/client.(*Conn).recv(0xc2080275f0)
    /home/scottjab/gocode/src/github.com/fluffle/goirc/client/connection.go:272 +0x3e
created by github.com/fluffle/goirc/client.(*Conn).postConnect
    /home/scottjab/gocode/src/github.com/fluffle/goirc/client/connection.go:241 +0x149

goroutine 23 [select]:
github.com/fluffle/goirc/client.(*Conn).runLoop(0xc2080275f0)
    /home/scottjab/gocode/src/github.com/fluffle/goirc/client/connection.go:314 +0xf9
created by github.com/fluffle/goirc/client.(*Conn).postConnect
    /home/scottjab/gocode/src/github.com/fluffle/goirc/client/connection.go:242 +0x161

goroutine 24 [select]:
github.com/fluffle/goirc/client.(*Conn).ping(0xc2080275f0)
    /home/scottjab/gocode/src/github.com/fluffle/goirc/client/connection.go:299 +0x220
created by github.com/fluffle/goirc/client.(*Conn).postConnect
    /home/scottjab/gocode/src/github.com/fluffle/goirc/client/connection.go:245 +0x1ad

goroutine 650 [runnable]:
net/http.(*persistConn).readLoop(0xc20803c4d0)
    /usr/lib/golang/src/pkg/net/http/transport.go:868 +0x829
created by net/http.(*Transport).dialConn
    /usr/lib/golang/src/pkg/net/http/transport.go:600 +0x93f

goroutine 651 [select]:
net/http.(*persistConn).writeLoop(0xc20803c4d0)
    /usr/lib/golang/src/pkg/net/http/transport.go:885 +0x38f
created by net/http.(*Transport).dialConn
    /usr/lib/golang/src/pkg/net/http/transport.go:601 +0x957

```

--- a/plugins/catgur.go
+++ b/plugins/catgur.go
@@ -20,19 +20,21 @@ func Handler(commands <-chan types.Command) {
 		"catt":     "CatPics",
 		"sitting":  "sittinglikehumans",
 		"tuckedin": "tuckedinkitties",
-		"bigcat": "bigcats",
-		"beard": "beardsandcats",
-		"bacon": "catswithbacon",
+		"bigcat":   "bigcats",
+		"beard":    "beardsandcats",
+		"bacon":    "catswithbacon",
 		"catbelly": "catbellies",
-		"tuxedo": "tuxedocats",
+		"tuxedo":   "tuxedocats",
 	}
 
 	for command := range commands {
 		var response = ""
 		if reddit, ok := cmds[command.Cmd]; ok {
 			response = randgur.RandomImageFromSubReddit(reddit)
-			log.Println(command.Line.Target(), ":", response)
-			command.Conn.Privmsg(command.Line.Target(), response)
+			if response != "" {
+				log.Println(command.Line.Target(), ":", response)
+				command.Conn.Privmsg(command.Line.Target(), response)
+			}
 		}
 	}
 }

--- a/plugins/randgur/randgur.go
+++ b/plugins/randgur/randgur.go
@@ -26,5 +26,8 @@ func RandomImageFromSubReddit(subreddit string) string {
 	if err != nil {
 		log.Fatal(err)
 	}
-	return results[randInt(0, len(results)-1)].Link
+	if len(results) > 0 {
+		return results[randInt(0, len(results)-1)].Link
+	}
+	return ""
 }

